pkg/kubectl/cmd/create_service.go

```diff
@@ -134,6 +134,7 @@ func NewCmdCreateServiceNodePort(f *cmdutil.Factory, cmdOut io.Writer) *cobra.Co
 	cmdutil.AddValidateFlags(cmd)
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.ServiceNodePortGeneratorV1Name)
+	cmd.Flags().Int32("node-port", 0, "Port used to expose the service on each node in a cluster.")
```

Comment:
Only 0-65k are actually valid. We specify the bit width in the API because it's a transport format, but that's not important for kubectl flags.